frontend/public/components/_masthead.scss

```diff
@@ -14,12 +14,30 @@
   @media(min-width: $grid-float-breakpoint) {
     height: $masthead-height-desktop;
   }
+  &__co-actions-menu {
+    padding: 0px 0px 0px 5px;
+  }
   &__console-picker {
     margin-left: 10px;
     @media(max-width: $grid-float-breakpoint-max) {
       display: none;
     }
   }
+  &__dropdowns {
+    display: flex;
+    justify-content: space-between;
+    margin-left: auto;
+    margin-right: 20px;
+  }
+  &__help {
```

Comment:
We will only want this to appear at desktop resolutions.  Should appear in the left nav for mobile.

Reply:
It should be in the left nav with Workloads and everything?

Reply:
Thanks, I'll hide it.